drivers/lora/sx1276.c

```diff
@@ -53,6 +53,8 @@ struct sx1276_data {
 	RadioEvents_t sx1276_event;
 	u8_t *rx_buf;
 	u8_t rx_len;
+	s16_t rx_rssi;
+	s8_t rx_snr;
```

Comment:
revert the order of the 2 attributes for the alignment to look obvious.

Reply:
Done

Reply:
no now you really broke the alignment. I meant to to so:

```
u8_t rx_len;
s8_t rx_snr;
s16_t rx_rssi;
```

Btw, rssi is always mesured on rx as well as snr, just call them rssi and snr, no need of the rx_ prefix.